plugins/MobileAppType/plugin.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "MobileAppType",
+  "description": "Allows you to create Mobile Apps"
```

Comment:
What about something like `Analytics for Mobile: lets you manage 'Mobile App' properties and creates an optimized perspective of your mobile data.`. The SDKs are rather irrelevant and what the type actually does is presenting the data in a different perspective by showing different reports, using different naming, etc.